app/src/marketplace/containers/ProductPage/Hero.jsx

```diff
@@ -43,7 +71,7 @@ const Hero = () => {
     const isDataUnion = !!(product && isDataUnionProduct(product))
     const isProductSubscriptionValid = useSelector(selectSubscriptionIsValid)
     const subscription = useSelector(selectContractSubscription)
-    const { account } = useWeb3Status()
+    const account = useAccountAddress()
```

Comment:
use `useAccountAddress` instead of `useWeb3Status` to avoid prompting for Metamask on page load.